configuration/concepts/nutrition.csv

```diff
@@ -0,0 +1,51 @@
+uuid,Void/Retire,Same as mappings,Fully specified name:en,Fully specified name:fr,Short name:en,Short name:fr,Description:en,Description:fr,Data class,Data type,Complex data handler,Answers,Members,Allow decimals,Units,Normal low,Normal high,Absolute low,Absolute high,_version:1,_order:1400
+edaef9f4-2b5b-4b71-9019-74f1d40ad4d7,,,Oedema,Œdème,Oedema,Œdème,,,Misc,N/A,,,,,,,,,,,
+1503f027-2693-4674-b658-de41d5df60e4,,,Weight-for-height<-3SD/BMI<16,PT <-3ET (IMC <16),Weight-for-height<-3SD/BMI<16,PT <-3ET (IMC <16),,,Misc,N/A,,,,,,,,,,,
+8dfc3ec9-771a-42b6-ad45-aa0ac15b660c,,,MUAC<115mm,PB<115mm,MUAC<115mm,PB<115mm,,,Misc,N/A,,,,,,,,,,,
+6a7ea608-0546-40f1-92d2-cd020e980414,,,Admission Criteria,Critère d'admission,Admission Criteria,Critère d'admission,,,Misc,Coded,,Oedema;Weight-for-height<-3SD/BMI<16;MUAC<115mm,,,,,,,,,
+a2224476-1827-4270-9ace-c8fd7898fd21,,,Readmission,Réadmission,Readmission,Réadmission,,,Misc,Coded,,Yes;No,,,,,,,,,
+fc1f3f86-9332-4714-a4f4-ce88c173a5c2,,,Weight goal,Objectif de poids,Weight goal,Objectif de poids,,,Misc,Numeric,,,,TRUE,Kg,,,,,,
+5f2ee7bc-8dcf-4617-b31c-2ac6c07617b5,,,0,0,0,0,,,Misc,N/A,,,,,,,,,,,
+e7c0fefc-96e8-4ccb-b24b-92ec2024f579,,CIEL:1362,+,+,+,+,,,Misc,N/A,,,,,,,,,,,

```

Comment:
Could you set the FSN as "One plus" and "Two plus"
in fr: "Un plus", "Deux plus"
and keep the short name as "+" and "++"

Reply:
Done